src/components/DiagramFrame/types.ts

```diff
@@ -0,0 +1,174 @@
+import { ILookmlModel, ILookmlModelExplore, IGitBranch } from "@looker/sdk/lib/sdk/4.0/models"
+import { SelectionInfoPacket, VisibleViewLookup } from "../interfaces"
+import { getActiveGitBranch, getAvailGitBranches, DiagramError } from "../../utils/fetchers"
+import { OVERRIDE_KEY_SUBTLE, DIAGRAM_HEADER_HEIGHT } from "../../utils/constants"
+import { DiagrammedModel } from "../../utils/diagrammer"
+import styled from "styled-components"
+import {
+  Aside,
+  Section,
+  SelectOptionProps,
+  Icon,
+  theme,
+  Header,
+} from "@looker/components"
+
+export const Italics = styled.span`
+  font-style: italic;
+`
+
+export const SettingsPanel = styled(Aside as any)`
```

Comment:
Mixing types and styled components again.

considering moving these to `DiagramFrame/components/SettingsPanel` etc